server/server.go

```diff
@@ -3349,13 +3363,19 @@ func getNextMsg(cs *stores.ChannelStore, nextSeq, lastSent *uint64) *pb.MsgProto
 	}
 }
 
-func (s *StanServer) getSequenceFromStartTime(cs *stores.ChannelStore, startTime int64) uint64 {
-	return cs.Msgs.GetSequenceFromTimestamp(startTime)
+func (s *StanServer) getSequenceFromStartTime(cs *stores.ChannelStore, startTime int64) (uint64, error) {
+	tm, err := cs.Msgs.GetSequenceFromTimestamp(startTime)
+	if err != nil {
+		s.log.Errorf("Unable to get sequence from start time: %v", err)
+		return 0, err
```

Comment:
I will, but the top layer may log something like "unable to add subscription: %v", and so this trace is adding more information: it happened when trying to get the sequence start. That message itself will contain the reason for the failure coming from the store. Unless we change from error to something that allows us to add causes (like a stack), we will always have this issue.